modules/007-registrypackages/images/drbd/werf.inc.yaml

```
@@ -35,3 +35,4 @@ shell:
     - chmod +x /install
     - chmod +x /uninstall
     - echo 'GIT-hash:123456' > /drbd/drbd/.drbd_git_revision
+    - sed 's/[[:space:]]curl[[:space:]]/ d8-curl -k /g' -i /drbd/drbd/drbd-kernel-compat/gen_compat_patch.sh
```

Hello! I think it is better to use sed with the -e option and use < and > as word boundaries, instead of [[:space:]]. In this case, the tab will not be removed from the last command curl in gen_compat_patch.sh script
```suggestion
    - sed -e "s/\<curl\>/d8-curl -k/g" -i /drbd/drbd/drbd-kernel-compat/gen_compat_patch.sh
```